scripts/pipelines/gcloud/pipeline_generator.sh

```diff
@@ -181,6 +181,17 @@ function checkOrUploadFlutterImage {
     fi
 }
 
+function addRoles {
+  echo -e "${green}Executing setup-principal-account.sh to give the necessary roles to the cloud build service agent for this pipeline.${white}"
+  gCloudProjectNumber="$(gcloud projects list | grep "$gCloudProject" | awk '{ print $NF }')"
```

Comment:
You can get easily the current project ID
```suggestion
  gCloudProjectID="$(gcloud config get-value project)"
```